dhis-2/dhis-web-api/src/main/resources/openapi/EnrollmentsExportController.md

```diff
@@ -53,73 +55,74 @@ excluded by default.
   are returned.
 - When `orgUnitMode=ALL`, `orgUnitMode=CAPTURE` or `orgUnitMode=ACCESSIBLE` the `orgUnits` parameter
   is not allowed.
 
 ### `*.parameter.EnrollmentRequestParams.orgUnitMode`
 
-Get enrollments using given organisation unit selection mode.
+Get enrollments using the given organisation unit selection mode.
 
-- When `orgUnitMode=SELECTED`, `orgUnitMode=CHILDREN` or `orgUnitMode=DESCENDANTS`, the `orgUnit`
-  parameter is
-  required to specify which enrollments to return.
-- When `orgUnitMode=ALL` enrollments will be downloaded irrespective of the organization unit they
-  are
-  owned by. To use this parameter, the user needs the `Search Tracked entity in all org units`
+- When `orgUnitMode=SELECTED`, `orgUnitMode=CHILDREN` or `orgUnitMode=DESCENDANTS`, the `orgUnits`
+  parameter is required to specify which enrollments to return.
+- When `orgUnitMode=ALL` enrollments will be returned irrespective of the organization unit they
+  are owned by. To use this parameter, the user needs the `Search Tracked entity in all org units`
   authority.
-- When `orgUnitMode=ACCESSIBLE` enrollments owned by any org unit in the users capture scope will be
-  returned.
-- When `orgUnitMode=CAPTURE` enrollments that has an enrollment org unit in the users capture scope
-  will
-  be returned.
+- When `orgUnitMode=ACCESSIBLE` enrollments owned by any organisation unit in the user's 
+  effective search scope will be returned. Effective search scope is the union of user's defined 
+  search scope and capture scope.
+- When `orgUnitMode=CAPTURE` enrollments that have an enrollment organisation unit in the user's 
+  capture scope will be returned.
 
 ### `*.parameter.EnrollmentRequestParams.program`
 
-Get enrollments enrolled in the given tracker program.
+Get enrollments within the specified tracker program.
 
 ### `*.parameter.EnrollmentRequestParams.status`
 
-Get enrollments in the given status.
+Get enrollments with the specified status.
 
 ### `*.parameter.EnrollmentRequestParams.programStatus`
 
-Get enrollments in the given status.
+Get enrollments with the given status.
 
 **DEPRECATED as of 2.42:** Use parameter `status` instead.
 
-See `status` for details.
+See `status` for details on valid statuses.
 
 ### `*.parameter.EnrollmentRequestParams.trackedEntityType`
 
-Get enrollments of tracked entities of the given type.
+Get enrollments of tracked entities of the specified tracked entity type. 
 
 ### `*.parameter.EnrollmentRequestParams.trackedEntity`
 
-Get enrollments of tracked entity with the given UID.
+Get enrollments of the tracked entity with the specified UID. 
 
 ### `*.parameter.EnrollmentRequestParams.updatedAfter`
 
-Get enrollments updated after the given date and time.
-This parameter is inclusive, so results with the exact date and time specified will be included in the response.
+Get enrollments updated after the given date and time. An enrollment is considered updated if 
```

Comment:
An enrollment is considered updated also if any field is updated like `enrolledAt`, right?
Should we mention it?